readthedocs/api/v3/serializers.py

```diff
@@ -0,0 +1,458 @@
+import datetime
+import urllib
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from rest_flex_fields import FlexFieldsModelSerializer
+from rest_flex_fields.serializers import FlexFieldsSerializerMixin
+from rest_framework import serializers
+
+from readthedocs.builds.models import Build, Version
+from readthedocs.projects.constants import LANGUAGES, PROGRAMMING_LANGUAGES
+from readthedocs.projects.models import Project
+
+
+class UserSerializer(FlexFieldsModelSerializer):
+
+    class Meta:
+        model = User
+        fields = [
+            'username',
+            'date_joined',
+            'last_login',
```

Comment:
Mmm... Not sure if this is really important.

In theory, you can see these info from people you share a project with. We already remove first and last name just in case, but those are bigger ones in case of leak.

Reply:
If we care about `last_login`, we should remove `date_joined` as well.

Reply:
GitHub does show the date_joinded field, last login feels like a more private thing

Reply:
I'm +1 on removing last login.